### PR TITLE
New CLI commands

### PR DESCRIPTION
**Pull request by intoeetive**
php eecli.php addons:list
php eecli.php addons:install
php eecli.php addons:update
php eecli.php addons:uninstall

### CODE REVIEW

File: docs/cli/built-in-commands/addons.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+[TOC]
+
+## `addons:list`
+
+Lists all add-ons available in the system.
+
+    php eecli.php addons:list
+
+List add-ons that can be installed:
```
Comment (TomJaeger) [inline docs/cli/built-in-commands/addons.md]: @matthewjohns0n is this correct? 

List installed lists add-ons that can be installed?  Not ones that are installed?

File: docs/cli/built-in-commands/addons.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+or
+
+    php eecli.php addons:list update-available
+
+List add-ons that can be uninstalled:
+
+    php eecli.php addons:list u
+or
+
+    php eecli.php addons:list uninstalled
```
Comment (TomJaeger) [inline docs/cli/built-in-commands/addons.md:34]: @matthewjohns0n same here?  List add-ons that can be uninstalled?  VS ones that are currently uninstalled.

I'm thinking the past tense doesn't make sense for for an action that's currently being taken.

Thoughts?

File: docs/cli/built-in-commands/backup-database.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Options list:
+
+    --relative_path=<value>
+    -p <value>
+        Path to database backup, relative to the cache folder
+
+    --absolute_path=<value>
+    -a <value>
+        Absolute path to database backup
```
Comment (TomJaeger) [inline docs/cli/built-in-commands/backup-database.md]: I assume this is path to where you want backup stored?

**Review by TomJaeger — COMMENTED** [review]
Please see comments below!

File: docs/cli/built-in-commands/addons.md (hunk 1 above)
Comment (matthewjohns0n) [reply docs/cli/built-in-commands/addons.md]: I see what you're saying - yes this needs to change. Good catch.

**Review by TomJaeger — APPROVED** [review]

**Pull request merged**